packages/connectivity/connectivity_platform_interface/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: A common platform interface for the connectivity plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/connectivity/connectivity_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 1.0.3
+version: 2.0.0
```

Comment:
We want to avoid breaking changes. Do you think we can return `null` when the plugin is unable to determine the status?

Reply:
Yes, I think that's another good option, but if we return `null`, our plugin will never be null-safe, right?

Reply:
(We discussed offline, and we're going with adding the value to the Enum!)